travel_platform/smart_ordering.py

```diff
@@ -0,0 +1,41 @@
+data = [{'name': 'Pavlo', 'age': 27, 'gender': 'male'},
```

Comment:
btw, this data is also not needed here. It may be just a part of unit tests, if needed.